<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_auto</class>
 <widget class="QWidget" name="edit_auto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>86</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_edit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>380</y>
     <width>86</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_poisk">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_poisk">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>86</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите гос номер авто</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_gosnomer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_vin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_ispraven">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>109</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Исправен</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_remont">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>109</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>На ремонте</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_marka_model">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Марка и модель</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Госномер</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_vin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>VIN</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
